dbt/scripts/utils/export.py

```diff
@@ -177,54 +177,73 @@ def export_models(
                 )
                 sheet.add_table(table)
 
+                # Parse column format settings by col index. Since column
+                # formatting needs to be applied at the cell level, we'll
+                # first parse all format settings for each column, and then
+                # we'll iterate every cell once to apply all formatting at the
+                # same time
+                column_format_by_index = {}
```

Comment:
The refactor in this file is more substantial than I was expecting, and we could probably have gotten away with a smaller change, but my motivation was to reduce the number of times we iterate the rows of the spreadsheet to apply formatting. By parsing the column formats ahead of time, we can apply them all in one pass, in contrast to the previous approach that required a separate pass for each format config.